### ISSUE TITLE
System-wide hitches when using TeamRedMiner with Adrenalin 21.10.1-21.10.3 on RX 6700 XT.

### ISSUE THREAD

**Issue by Rajeh16** [todxx/teamredminer#451]
I recently tried updating from Adrenalin 21.8.2 to 21.10.3 and found that when browsing the web while mining I would get second long hitches at random, regardless of intensity or mining mode. I haven't tested 21.9.x.

I've downgraded to 21.8.2 for the meantime. I hope this gets fixed because the newer drivers have some bug fixes that would be beneficial to me.

**Comment by Flufflol** [comment]
Same issue here, started happening ever since 21.10.1 and it persisted through 21.10.2 and 21.10.3. Today I upgraded to 21.10.4 and same issue. This did not happen when i was using 21.9.2 so I decided to downgrade to that and voilla, no more hitches.

Hope this get fixed so I can use latest drivers, not sure if it is from TRM or AMD Drivers, but something is definitely wrong considering it does not happen in 21.9.2.

6700 XT here aswell.

**Comment by jimiche** [comment]
Tested with TRM v0.8.6.3 and new AMD Drivers 21.11.1, same issue unfortunatly.

**Comment by Flufflol** [comment]
> Tested with TRM v0.8.6.3 and new AMD Drivers 21.11.1, same issue unfortunatly.
> 
> EDIT: Runnning GMiner 2.70 now with AMD Drivers 21.11.1 without any issues.

So it is a TRM issue after all with the newest drivers, good to know. Hope they fix it soon.

**Comment by Flufflol** [comment]
Any plans of fixing this Issue? It is annoying for people that actually use their computer/rigs for something else besides purely mining, In my Case I use it for work while mining at the same time. Like I said previously, this did not happen with 21.9.2 drivers, ever since 21.10.1 it started happening and it is still happening to this day with newest Drivers.

I tested GMiner aswell and it does not have any issue, so it's clearly a TeamRedMiner issue
Please fix it.

**Comment by jimiche** [comment]
More than a month since issue was reported and the is not even an aknowledgment. I don't think they will prioritize on issues for people that is only running 1 gpu like us unfortunatly.

**Comment by Rajeh16** [comment]
> More than a month since issue was reported and the is not even an aknowledgment. I don't think they will prioritize on issues for people that is only running 1 gpu like us unfortunatly.

Yeah it's not looking great. I've just been sitting on 21.8.2 for now since I haven't had a need to update, but I'm sure I will have to eventually.

**Comment by Flufflol** [comment]
Same issue with the TRM 0.8.7 version that came out today.
Hope it will be fixed at some point, we small miners have feelings too :(

**Comment by tiagoa77** [comment]
Same issue with the TRM 0.9.0 :/

**Comment by jimiche** [comment]
No news? :(

**Comment by Flufflol** [comment]
This is still an issue even now with latest TRM 0.9.4.2 and Latest AMD Drivers 22.4.1.
Hopefully it will be fixed at some point, anything that requires GPU Acceleration seems to be freezing the computer for a few seconds, i'ts been like that ever since AMD 21.10.1 drivers, other miners don't have this problem so I am not sure what is wrong with TRM and the newest AMD drivers.

But please, some people do use their computers while mining to do some light stuff or play old games while doing so.
It wasn't a problem before AMD 21.10.1 drivers so hopefully it's gonna be fixed at some point so i can go back to TRM.
